renovate-config.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "timezone": "Europe/Warsaw",
+  "extends": ["config:base", ":disableDependencyDashboard"],
+  "schedule": [
+    "before 5am on Monday"
+  ],
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["*"],
+      "matchUpdateTypes": ["patch"],
+      "groupName": "all non-major dependency bump",
+      "groupSlug": "all-patch",
+      "automerge": true
+    },
+    {
+      "extends": "packages:linters",
+      "groupName": "linters",
+      "automerge": true
+    },
+    {
+      "extends": "packages:postcss",
+      "groupName": "postcss packages"
+    },
+    {
+      "extends": "packages:test",
+      "groupName": "test packages",
+      "automerge": true
+    },
```

Would it be wise to consider special treatment for testing packages, such as `postcss` and linters? A major bump could potentially disrupt our CI. My recommendation is to create a separate configuration for major upgrades and to have each one in a separate PR with automerging.

```suggestion
    {
      "matchUpdateTypes": [
        "major"
      ],
      "matchDepTypes": [
        "dependencies",
        "devDependencies"
      ],
      "automerge": false,
      "schedule": [
        "after 2am and before 3am"
      ]
    },
```

---

Linters should go together as they're really tightly-coupled together and it's easier to work with all-linter-related-packages update on a single branch.
Also, linters and tests are special cases -> even if we get a major bump of linter/test package it can be safely automerged as long as the CI passes. We don't want to even look at those updates as we don't have to